video_xblock/static/js/studio-edit/transcripts-manual-upload.js

```diff
@@ -132,20 +132,20 @@ function validateTranscriptFile(event, fieldName, filename, $fileUploader) {
  */
 function pushTranscript(lang, label, url, source, oldLang, transcriptsValue) {
     'use strict';
-    var indexLanguage;
+    var languageIndex;
     var i;
     for (i = 0; i < transcriptsValue.length; i++) {
```

Comment:
Since we're already here, can you please replace this for-loop with proper use of `Array.prototype.findIndex()`?